tests/test_main/test_anonymize_cli.py

```diff
@@ -0,0 +1,41 @@
+import runpy
+import shutil
+import sys
+from pathlib import Path
+from typing import Final
+
+import pydicom
+import pytest
+
+from dicom_utils.anonymize import is_anonymized
+
+
+num_dicom_test_files: Final[int] = 3
```

Comment:
I'm not always consistent about doing this, but I think PEP standards say that this should be in all caps.